boilerplate/app/screens/DemoShowroomScreen/DrawerIconButton.tsx

```diff
@@ -3,7 +3,8 @@ import { Pressable, PressableProps, ViewStyle, Platform } from "react-native"
 import Animated, { interpolate, interpolateColor, useAnimatedStyle } from "react-native-reanimated"
 import { useDrawerProgress } from "react-native-drawer-layout"
 import { isRTL } from "../../i18n"
-import { colors, spacing } from "../../theme"
+import { useAppTheme } from "app/utils/useAppTheme"
+import type { ThemedStyle } from "app/theme"
```

Comment:
This icon actually isn't consistently updating with the rest of the app. Haven't looked at why yet. note it took a few attempts to reproduce, but eventually the icon fails to turn white when moving into dark mode (that's the only way I've seen it fail so far, I've never seen it fail to turn dark when moving into light mode).

https://github.com/user-attachments/assets/80e8f84d-ab78-49b7-82c4-42c0d4de51d3

Reply:
I haven't been able to replicate this myself. The animated styles are using the colors returned by the useAppTheme() to determine the color of the bars.